core/types/rollup_cost.go

```diff
@@ -25,21 +26,34 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+const (
+	// offsets of the packed fields within the L1FeeScalarsSlot
+	BasefeeSlotOffset     = 0
+	BlobBasefeeSlotOffset = 4
+)
```

Comment:
These are the other way around in the current `storageLayout/L1Block.json` in the L2 contract updates PR

Reply:
I may be terribly confused, but my understanding is the offsets in storage layout are from the right-most end of the 32 byte slot. E.g. here's storage slot 0 of L1 block, which contains "number" and "timestamp", clearly left-zero-padded:

```
>> cast storage --rpc-url https://mainnet.base.org/ 0x4200000000000000000000000000000000000015 0
0x00000000000000000000000000000000000000006598973f00000000012111d0
```

You can see the block `number` there appears at bytes [24:32], meanwhile `storagelayout/L1Block.json` claims `number` has offset 0. 

```
 {
    "bytes": "8",
    "label": "number",
    "offset": 0,
    "slot": "0",
    "type": "uint64"
  },
  ```

Reply:
Following up from our slack discussion, I have made the offset values here match those in `storageLayout/L1Block.json` and now do the appropriate offset translation in code after doing the read.